enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -0,0 +1,302 @@
+    | `Skip`        | (unused)                                                                                                                                              |
+
+- **Snippet**
+
+  ```golang
+  type PlacementBinding struct {
+    metav1.TypeMeta   `json:",inline"`
+    metav1.ObjectMeta `json:"metadata,omitempty"`
+    // +kubebuilder:validation:Optional
+    BindingOverrides BindingOverrides `json:"bindingOverrides,omitempty"`
+    // +kubebuilder:validation:Optional
+    RolloutStrategy clusterv1alpha1.RolloutStrategy `json:"rolloutStrategy,omitempty"` // <-- New field
+    // This field provides the ability to select a subset of bound clusters
+    // +kubebuilder:validation:Optional
+    // +kubebuilder:validation:Enum=restricted
+    SubFilter SubFilter `json:"subFilter,omitempty"`
+    // +kubebuilder:validation:Required
+    PlacementRef PlacementSubject `json:"placementRef"`
+    // +kubebuilder:validation:Required
+    // +kubebuilder:validation:MinItems=1
+    Subjects []Subject              `json:"subjects"`
+    Status   PlacementBindingStatus `json:"status,omitempty"`
+  }
+  ```
+
+### Implementation Details/Notes/Constraints
+
+For the `Placement` library, this requires importing at least this package version (the `Placement` library is in the
+`v1alpha1` version):
+
+```
+  open-cluster-management.io/api v0.11.1-0.20230828015110-b39eb9026c6e
+```
+
+For testing, the `governance-policy-propagator` doesn't currently account for multiple managed clusters. As part of this
+enhancement, the test flows would need to be enhanced (and/or a separate workflow created) that deploys multiple managed
+clusters.
+
+The `PlacementDecisionGetter` returns an array of pointers (`[]*clusterv1beta1.PlacementDecision`) because it was
+intended to retrieve from a cache, so the implementation could consider setting up a `PlacementDecision` cache instead
+of using the Kubernetes client directly.
+
+### Risks and Mitigation
+
+- The `Placement` library is relatively new and untested outside of its repo and this implementation leans heavily on
+  its logic. While it works in theory, there could be some tweaks/adjustments as development proceeds, lengthening time
+  for implementation. The phased approach intends to address this to make partial implementation feasible.
+
+## Design Details
+
+### Open Questions
+
+1. Should the rollout handler be instantiated with a dynamic watcher/cache instead of the controller client?
+2. Should `RolloutStatus` be reflected per-cluster in the root policy? (The rollout status would also be reflected in
+   each replicated policy.)
+3. Should the `RolloutStrategy` be implemented in the `Policy` CRD instead of `PlacementBinding`? (If that were the
+   case, it would also need to be added to `PolicySet`.)
+4. How should a policy handle when placement decision groups shift? (This implementation leans on the `Placement`
+   library to handle such shifts, only handling the clusters that it returns.)
```

Comment:
Does this risk clusters being skipped? Or gaps in the decision groups if clusters are unbound/rebound?